pkg/capr/planner/planner.go

```diff
@@ -855,6 +878,9 @@ func (p *Planner) reconcile(controlPlane *rkev1.RKEControlPlane, tokensSecret pl
 		if err != nil {
 			return err
 		}
+		if resetFailureCountOnSystemAgentRestart {
+			plan.ResetFailureCountOnSystemAgentRestart = true
+		}
```

Comment:
```suggestion
		plan.ResetFailureCountOnSystemAgentRestart = resetFailureCountOnSystemAgentRestart
```

Reply:
Good point, the conditional was more useful during testing